packages/mdc-chips/chip/index.js

```diff
@@ -69,8 +79,35 @@ class MDCChip extends MDCComponent {
     }
   }
 
+  initialSyncWithDOM() {
+    this.handleInteraction_ = (evt) => this.foundation_.handleInteraction(evt);
+    this.handleTransitionEnd_ = (evt) => this.foundation_.handleTransitionEnd(evt);
+    this.handleTrailingIconInteraction_ = (evt) => this.foundation_.handleTrailingIconInteraction(evt);
+
+    ['click', 'keydown'].forEach((evtType) => {
```

Comment:
I think making this array and the one below into variables will read better, and then you can share it in the destroy.

Reply:
Done! I'm not sure why we omitted touch events from the first array...so I added them in and used the same array for chip interaction and trailing icon interaction.

Reply:
> I'm not sure why we omitted touch events from the first array

Maybe because the code in `handleInteraction` is a no-op for anything besides click or keyboard events anyway? I don't think adding touch events will be harmful but they'll also never do anything.

Reply:
Oh good point. The touch events are also a no-op in `handleTrailingIconInteraction`, so I'm just gonna remove them altogether.